CHANGELOG.md

```diff
@@ -5,58 +5,6 @@
 
 > All notable changes to this project will be documented in this file
 
-## [1.80.0-beta.1](https://github.com/open-sauced/insights/compare/v1.79.3-beta.1...v1.80.0-beta.1) (2023-12-13)
```

Comment:
Looks like there was a merge from beta issue. This file shouldn't be a part of the PR.